Python debug configuration is added

SamPosh opened the pull request:
Python debug DAP configuration is added for handlers.

This sample can be helpful for beginners who wanted to debug python.

I spent 2 weeks to figure it out. Thought of adding the sample here so that others need not to spend the same amount of time. 

Thank you so much for this kickstart.nvim .

feoh commented:
Hi, thanks for the contribution but I have very mixed feelings about this one.

This isn't how I configure nvim-dap-python, and I think a single example will suffice just fine.

I'm tempted to close this unless you can make a super compelling argument for your change :)

SamPosh commented:
I have added pytest debug configuration and profiler debug configuration as well.
For a beginner there will be confusion where to configure what , so I added it.

If you think these additional configurations can be added in much simpler way then it would be good if you can add those examples (I also will learn from it) :)

feoh commented:
I can appreciate that. The problem is that there are a million different languages and everyone has different favorites. If we add them all kickstart will become ENORMOUS and impossible to maintain.

Here's a [link](https://github.com/feoh/git_dotfiles/blob/master/.config/nvim/lua/kickstart/plugins/debug.lua) to my debug configuration that might give you some ideas, but I'm closing this for now. Thanks again for the contribution!
